client/tests/integration_tests/add_asset.rs

```diff
@@ -115,7 +115,9 @@ fn client_add_asset_with_decimal_should_increase_asset_amount() -> Result<()> {
         )),
     );
     // and check that it is added without errors
-    let sum = quantity.checked_add(quantity2)?;
+    let sum = quantity
+        .checked_add(quantity2)
+        .map_err(|e| eyre::eyre!("{}", e))?;
```

Comment:
Doesn't this just wraps it into `eyre::Report` twice?

Reply:
Nope. Turning `FixedPointOperationError`, (which doesn't `impl std::error::Error`, because of `#[no_std]`), into `eyre::Report`.